src/engine/renderer/tr_image.cpp

```diff
@@ -1653,6 +1653,7 @@ static void R_LoadImage( const char **buffer, byte **pic, int *width, int *heigh
 
 	ext = COM_GetExtension( filename );
 
+	// the Daemon's default strategy is to use the hardcoded path if exists
```

Comment:
Note that I don't like this strategy but I remember people prefer this. Basically I don't like the behaviour of loading the hardcoded path if exists because it means a pak shipping `.crn` files alternatives to `.tga` ones will never seen the `.crn` files used if an older pak has the `.tga` one, defeating the idea of « shipping better formats », but when I asked to remove that part and only rely on a « best strategy », some people had strong arguments at this time. Basically, id Software did the exact opposite thing with `dds` files for Doom3, and Xonotic guys did the exact opposite thing with Darkplaces (hence the current patch).

It was told people can hardcode stuff if they want to force a format, and use extension independent path if they want to never force it. But in case of some uncommon (or now uncommon) formats like md3, it means editing a binary file to make the path ext independent.